spec/acceptance/sensu_asset_spec.rb

```diff
@@ -31,9 +36,12 @@
       pp = <<-EOS
       include ::sensu::backend
       sensu_asset { 'test':
-        url     => 'http://example.com/asset/example.tar',
-        sha512  => '4f926bf4328fbad2b9cac873d117f771914f4b837c9c85584c38ccf55a3ef3c2e8d154812246e5dda4a87450576b2c58ad9ab40c9e2edc31b288d066b195b21b',
-        filters => ['System.OS==windows'],
+        url      => 'http://example.com/asset/example.tar',
```

Comment:
should this be zip to match the content type?

Reply:
Technically, yes, but I wanted to test a modification to `metadata`.  I'll update to have file type match content type.

Reply:
Please do and squash/rebase commits. Thanks!